runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -169,6 +181,25 @@ private void renderRockToMine(Graphics2D graphics)
 		}
 	}
 
+	private void renderRockToMineAtMaP(Graphics2D graphics)
+	{
+		GameObject glowingRock = util.getGlowingRock();
+
+		if (glowingRock != null)
+		{
+			Point rockPoint = glowingRock.getMinimapLocation();
```

Comment:
`TileObject::getMinimapLocation` is nullable, and `OverlayUtil::renderMinimapLocation` isn't null safe

Reply:
K added null check.

Reply:
if (rockPoint!=null) Will do?